apps/web/pages/event-types/index.tsx

```diff
@@ -126,11 +128,17 @@ const Item = ({ type, group, readOnly }: { type: EventType; group: EventTypeGrou
           data-testid={"event-type-title-" + type.id}>
           {type.title}
         </span>
-        <small
-          className="hidden font-normal leading-4 text-gray-600 sm:inline"
-          data-testid={"event-type-slug-" + type.id}>{`/${group.profile.slug}/${type.slug}`}</small>
+        {group.profile.slug ? (
+          <small
+            className="hidden font-normal leading-4 text-gray-600 sm:inline"
+            data-testid={"event-type-slug-" + type.id}>
+            {`/${group.profile.slug}/${type.slug}`}
+          </small>
+        ) : (
+          <UpgradeTeamsBadge />
```

Comment:
@zomars can we give this a props => upgradeLink={true} that uses the "upgrade" stripe api link vs the /team url

Reply:
keep the logic with tooltip etc but link to the stripe checkout

Reply:
No need to prop AFAIK. we can encapsulate that logic in the components itself.

Reply:
not sure what that means but if you have a free minute may you wanna add that? you built the initial "upgrade here" top bar banner